salt/proxy/philips_hue.py

```diff
@@ -0,0 +1,483 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2015 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''
+Philips HUE lamps module for proxy.
+'''
+
+from __future__ import absolute_import
+
+# Import python libs
+import logging
+import requests
```

Comment:
Ok. You'll need to try/except that import for now since requests is not a hard dependency.